demos/rag_agentic/notebooks/Level2_simple_agentic_with_websearch.ipynb

```diff
@@ -56,7 +50,7 @@
   },
```

Comment:
`Next, we will initialize our environment as described in detail in our [\"Getting Started\" notebook](demos/rag_agentic/notebooks/Level0_getting_started_with_Llama_Stack.ipynb). Please refer to it for additional explanations.` 

This link is not working. I think the issue was present prior to this PR.